javascript/src/methods.js

```diff
@@ -442,9 +439,15 @@ methods.addPolylines = function(polygons, layerId, group, options, popup, popupO
       .cbind(options);
 
     addLayers(this, "shape", df, function(df, i) {
-      let shape = df.get(i, "shapes")[0];
-      shape = HTMLWidgets.dataframeToD3(shape);
-      return L.polyline(shape, df.get(i));
+      let shapes = df.get(i, "shapes");
```

Comment:
Haha I did use `map` first and then noticed the for loop in `addPolygons`, so I changed it to for loop to keep it consistent.